asistente_ladm_col/gui/qgis_model_baker/dlg_import_schema.py

```diff
@@ -215,18 +215,15 @@ def get_checked_models(self):
         return checked_models
 
     def show_settings(self):
-        dlg = SettingsDialog(qgis_utils=self.qgis_utils, conn_manager=self.conn_manager, db_source=self.db_source)
+        # We only need those tabs related to Model Baker/ili2db operations
+        tab_pages_list = [SETTINGS_CONNECTION_TAB_INDEX, SETTINGS_MODELS_TAB_INDEX]
+        dlg = SettingsDialog(qgis_utils=self.qgis_utils, conn_manager=self.conn_manager, db_source=self.db_source, tab_pages_list=tab_pages_list)
```

Comment:
To discuss whether it is better as a setter...

Reply:
Required models are defined in the class constructor.

Reply:
Resolution: passed now as class parameters.